wpilibc/src/main/native/include/frc/Compressor.h

```diff
@@ -113,34 +121,65 @@ class Compressor : public wpi::Sendable,
   void Disable();
 
   /**
-   * Enable compressor closed loop control using digital input.
+   * Enables the compressor in digital mode. The compressor will turn on when
+   * the pressure switch indicates that the system is not full, and will turn
```

Comment:
Mention that this mode is control based on the digital pressure switch (rather than the analog one on a PH)